miden/src/cli/verify.rs

```diff
@@ -17,33 +18,39 @@ pub struct VerifyCmd {
     output_file: Option<PathBuf>,
     /// Path to proof file
     #[clap(short = 'p', long = "proof", value_parser)]
-    proof_file: PathBuf,
+    proof_file: Option<PathBuf>,
```

Comment:
I would keep `proof_file` as required and try to infer input/output files based on it (rather than using input file to infer proof/output file).